recipe/meta.yaml

```diff
@@ -12,22 +12,23 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  skip: True  # [py<36]
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
+    - numpy {{ numpy }}
   run:
     - python >=3.6
```

Comment:
Now that we've dropped `noarch: python`, we don't need `>=3.6` in the `run` section. That limit should be automatically added by the `python` from the `host` section.

Reply:
Thanks Cheng! I will go ahead and update this. :)